cadctl/cmd/investigate/investigate.go

```diff
@@ -142,6 +143,19 @@ func getInvestigation(alertTitle string) *investigation.Investigation {
 	} else if strings.Contains(alertTitle, "ClusterProvisioningDelay -") {
 		return investigation.NewInvestigation(cpd.Investigate, "ClusterProvisioningDelay")
 	}
+
+	// Return early if experimental features are not enabled
+	experimentalFeatures, experimentalExists := os.LookupEnv("CAD_EXPERIMENTAL_ENABLED")
+	if !experimentalExists || strings.ToUpper(experimentalFeatures) != "TRUE" {
```

Comment:
Minor: Because we don't actually care about the distinction between `CAD_EXPERIMENTAL_ENABLED` being unset and empty, you just use `os.Getenv()` and simplify this statement to:
```
if strings.ToUpper(os.Getenv("CAD_EXPERIMENTAL_ENABLED")) != "TRUE" {
...
```
[os.Getenv()](https://pkg.go.dev/os#Getenv)

Reply:
Thanks!